composer.json

```diff
@@ -9,7 +9,7 @@
   "license": "MIT",
   "require": {
     "php": "^7.3 || ^8.0",
-    "sylius/sylius": "~1.8.0 || ^1.9.0 || ^1.10.0"
+    "sylius/sylius": "^1.9.0 || ^1.10.0"
```

Comment:
I think it should be: `~1.9.0 || ~1.10.0`, so it won't make any issues in the future when Sylius release the 1.11 version

Reply:
Of course, thank you.